Fun Commands/Mini-Games/Dyno-RPG/treat.txt

```diff
@@ -0,0 +1,8 @@
+{delete}
+{silent}
+{!role $1 -Infected}
```

Comment:
Is this a command what you want to ensure can only be used by someone who isn't infected?